src/models/Studio.ts

```diff
@@ -44,9 +44,13 @@ export function start(filePath: string, port: number = DEFAULT_PORT): void {
     }
   });
 
+  const studioBuildFolder = existsSync(resolve(__dirname, '../../node_modules/@asyncapi/studio/build')) ?
+    resolve(__dirname, '../../node_modules/@asyncapi/studio/build') : 
+    resolve(__dirname, '../../../../@asyncapi/studio/build');
```

Comment:
```suggestion
  const studioBuildFolder = require.resolve('@asyncapi/studio/build/index.html');
```

this should be universal solution. Works on my side, npm user. 
wdyt?

Reply:
@to-kr

Reply:
@to-kr do you plan to continue with this one?

Reply:
Sorry, I didn't catch your previous answer. Your solution looks good. Please fix it yourself.